### ISSUE TITLE
Introduce partial fit steps.

### ISSUE THREAD

**Issue by bcwein** [bcwein/DAT540-EvolutionaryANN#26]
This is an alternative approach to selecting agents is to introduce partial fit steps.  What we have done until now is to randomly create weights and biases and select the ones that work well. This is a quite stochastic approach. 

Partial fit will allow us to train the neural network on the experiences that are good and use this to generate new agents.

See: https://towardsdatascience.com/reinforcement-learning-q-learning-with-decision-trees-ecb1215d9131

**Comment by bcwein** [comment]
https://towardsdatascience.com/reinforcement-learning-without-gradients-evolving-agents-using-genetic-algorithms-8685817d84f

**Comment by bcwein** [comment]
Partial fit is done, but i struggle to get good results from the trained neural network.

It seesm that the partial fit does not give the trained neural network weights that are similar to the agent that performs well.

**Comment by bcwein** [comment]
Closing this issue due to this not yieldind good results and not becoming a main focus in our project.
